cog.c

```diff
@@ -442,7 +448,17 @@ main (int argc, char *argv[])
         g_set_application_name ("Cog");
     }
 
-    g_autoptr(GApplication) app = G_APPLICATION (cog_launcher_get_default ());
+    // We need to check whether we'll use automation mode before creating the launcher
+    gboolean automated = FALSE;
+    for (int i=0; i < argc; i++) {
```

Comment:
Nit: spacing around `i=0`. Also, it could start from 1 since [0] is the program name (as leveraged just above).